Source/Tools/FEXConfig/main.qml

```diff
@@ -601,6 +601,11 @@ ApplicationWindow {
                         text: qsTr("Enable non-tearing split-lock atomics")
                         config: "StrictInProcessSplitLocks"
                     }
+                    ConfigCheckBox {
+                        topPadding: 4
+                        text: qsTr("Enable PE volatile metadata")
+                        config: "VolatileMetadata"
+                    }
```

Comment:
This won't do anything for vanilla FEX, will it? FEXConfig isn't typically used with WoA FEX, so I think we can do without a dedicated option (the Advanced tab will still list it anyway).

Reply:
FEXconfig just works the same with windows fex, not sure why it shouldn't be here

Reply:
The setting doesn't have any effect on other (currently more commonly used) configurations. Options that don't do anything aren't good UX, but if you can find a way to disable it just for those more common configurations, I'd be fine with that as well.

Reply:
The same config file is used for both - it's not really something that can be distinguished. I can add a note that it applies to wine fex only? Considering it's now distributed via officials repos I don't think it makes sense to just hide it overall when FEXConfig really isn't that user facing anyway

Reply:
Fair, you've convinced me :)

FTR I also argued against adding a GUI for non-tearing split lock emulation. But it's a good point that the same config file may be shared for either type of FEX build, and it's definitely a useful setting for WoA configurations (even more so if distros are already shipping that widely).